File: Day6.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
def quebra(wordsSet):
    for x in wordsSet :
        auxLine = x
        even = ""
        odd = ""
        #print ('Case: ', x)
        for i in range (0,len(auxLine)) :
            if i%2 == 0:
                even += auxLine[i]
            else:
                odd += auxLine[i]
        print(even, odd)

arrayList = []
readNLines = int(input())
while readNLines > 0 :
    line = input()
    if (line):
        arrayList.append(line)
    readNLines -= 1
quebra(arrayList[:])

File: Day6.js
function processData(input) {
    //Enter your code here
    var evenIndexed = "";
    var oddIndexed = "";

    for (i=0; i < input.length - 2;){
        evenIndexed += input.charAt(i);
        oddIndexed += input.charAt(i+1);
        i = i+2;
    }
    console.log(evenIndexed + " " + oddIndexed + '\n');
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});
